features/makepot.feature

```diff
@@ -2458,3 +2458,391 @@ Feature: Generate a POT file of a WordPress project
       Success: POT file successfully generated!
       """
     And the contents of the result.pot file should match /^msgid/
+
+  Scenario: Extract strings from block.json files
+    Given an empty foo-plugin directory
+    And a foo-plugin/foo-plugin.php file:
+      """
+      <?php
+      /**
+       * Plugin Name: Foo Plugin
+       */
+      """
+    And a foo-plugin/block.json file:
+      """
+      {
+        "name": "my-plugin/notice",
+        "title": "Notice",
+        "category": "common",
+        "parent": [ "core/group" ],
+        "icon": "star",
+        "description": "Shows warning, error or success notices  ...",
+        "keywords": [ "alert", "message" ],
+        "textDomain": "foo-plugin",
```

Comment:
I discovered today that `textdomain` is the preferred way to phrase it in WordPress core, see my related comment: https://core.trac.wordpress.org/ticket/48529#comment:15

Do you think we should leave it as is or it makes sense to update?